plugin/trino-elasticsearch/pom.xml

```diff
@@ -427,4 +444,47 @@
             </plugins>
         </pluginManagement>
     </build>
+
+    <profiles>
+        <profile>
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <excludes>
+                                <!-- This test must be run in isolation  -->
+                                <exclude>**/TestElasticsearchBackpressure*</exclude>
```

Comment:
can you help me understand why is that?

Reply:
Statistics sometimes returned 0 as if they were not being updated. Apparently if more than one stats bean gets registered in the JVM, it can lead to stats not being updated properly. The test @kokosing mentioned as an example had a similar kind of isolation in place, presumably to avoid multiple stats bean registrations.

Reply:
https://github.com/trinodb/trino/pull/8379#discussion_r659003510 is also related

@MiguelWeezardo can you make sure your wisdom is retained as a code comment here in pom.xml?

Reply:
https://github.com/trinodb/trino/pull/8539